front-end/src/components/TranscriptUploadModal.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import Modal from 'react-modal';
+
+class TranscriptUploadModal extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      file: null,
+    };
+  }
+  
+  handleFile(e) {
+    console.log(e.target.files, "File Uploaded");
+    this.setState({file: e.target.files})
+  }
+
+  handleUpload(e) {
+    let file=this.state.file
+    let formData= new FormData()
+
+    formData.append('transcript', file)
+
+    axios({
+      url: '/some/api',
+      method: "PUT",
+      headers:{
+        authorization: 'token'
+      },
+      data: formData
+    })
+  }
```

Comment:
All axios handling should be done in a separate folder (see the previous "services" we had). We can make this a separate ticket if you want, but instead of using this axios function, `import TranscriptAPI from  "../services/TranscriptAPI";` and then call `TranscriptAPI.post({ data: formData });`.

Reply:
I'm good with making this another ticket 👍

Reply:
PR up now https://github.com/BestFloww/BestFlow/pull/20